server/services/store/sqlstore/public_methods.go

```diff
@@ -229,6 +239,27 @@ func (s *SQLStore) RefreshSession(session *model.Session) error {
 
 }
 
+func (s *SQLStore) ReplaceBlockID(currentID string, newID string, workspaceID string) error {
```

Comment:
This are still here, probably you need to regenerate the public API

Reply:
Arg, right!

Reply:
Addressed in 6d349c8